templates/endpoints_v2/examples/function_calling/DeployFunctionCalling.ipynb

```diff
@@ -19,17 +19,15 @@
     "You can use the `generate_config.py` script to generate a model configuration YAML for function calling. \n",
     "\n",
     "These are the fields that will be set in the resulting model YAMLs for function calling -\n",
-    "1. `enable_json_logits_processors: true` under `engine_kwargs`\n",
+    "1. Environment variables under `runtime_env`\n",
```

Comment:
is this really how we are enabling it? Definitely needs to be cleaned up - we should add this to list of tasks @shrekris-anyscale

Reply:
Yes it is. @kouroshHakha do we already have a plan to clean this up?